WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartMySitePrompts.kt

```diff
@@ -43,10 +44,16 @@ enum class QuickStartMySitePrompts constructor(
     ),
     PUBLISH_POST_TUTORIAL(
             QuickStartTask.PUBLISH_POST,
-            R.id.root_view_main,
-            R.id.bottom_nav_new_post_button,
-            R.string.quick_start_dialog_publish_post_message_short,
-            R.drawable.ic_create_white_24dp
+            if (BuildConfig.INFORMATION_ARCHITECTURE_AVAILABLE) R.id.my_site_scroll_view_root else R.id.root_view_main,
```

Comment:
It might be little more readable to create a new enum item (maybe rename this one to `OLD_PUBLISH_POST_TUTORIAL`?).

Reply:
AFAIU we get the `QuickStartMySitePrompts` by `QuickStartTask` but I think we would need to introduce some extra logic (not needed when removing the FF) in managing how we scroll the list of tutorials in `QuickStartUtils.getNextUncompletedQuickStartTask`. So if you agree in this case I would sacrifice the readability a bit (since it's time boxed to when we remove the FF) and not change the logic code. Let me know wdyt 😊.

Reply:
Makes sense 👍